invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.variables

```diff
@@ -36,8 +36,5 @@
 
 /*--- Fonts ---*/
 
-@importGoogleFonts : false;
```

Comment:
I think we still need this one, to prevent loading fonts from Google (due to GDPR).